be/src/exprs/block_bloom_filter.hpp

```diff
@@ -44,8 +47,8 @@ class BlockBloomFilter {
     Status init(int log_space_bytes, uint32_t hash_seed);
     // Initialize the BlockBloomFilter from a populated "directory" structure.
     // Useful for initializing the BlockBloomFilter by de-serializing a custom protobuf message.
-    Status init_from_directory(int log_space_bytes, const Slice& directory, bool always_false,
-                               uint32_t hash_seed);
+    Status init_from_directory(int log_space_bytes, butil::IOBufAsZeroCopyInputStream* data,
+                               const size_t& data_size, bool always_false, uint32_t hash_seed);
```

Comment:
Using `const size_t` is enough.
```suggestion
                               const size_t data_size, bool always_false, uint32_t hash_seed);
```